### ISSUE TITLE
For data preprocessing scripts

### ISSUE THREAD

**Issue by markanother** [cjx96/DisenCDR#2]
Hi,

Would you please share the scripts for preprocessing the raw datasets (Amazon csv files)? I am confused about "filter out cold-start item entry in the test set from BiTGCF" in your paper. What do you mean by "cold-start item"? BiTGCF paper states that they retain the items with number of ratings greater than 10. Thank you very much.

**Comment by markanother** [comment]
@ cjx96

**Comment by markanother** [comment]
@cjx96

**Comment by cjx96** [comment]
Hi, for this issue, we find some items are cold-start in the test set of original Bi-TGCF preprocessed datasets.

(cold-start means the item does not show in the training set).

Therefore, we filter such entries in our versions.

**Comment by markanother** [comment]
Thank you for your reply.
